eve/workers/pod-builder/pod.yaml

```diff
@@ -34,12 +34,8 @@ spec:
       securityContext:
         privileged: true
       volumeMounts:
-        - name: docker-storage
-          mountPath: /var/lib/docker
         - name: worker-workspace
           mountPath: /home/eve/workspace
   volumes:
-    - name: docker-storage
```

Comment:
This mount is indeed needed, otherwise you write all the images data in the docker overlay and not in a proper volume and loose some performance. NOT RECOMENDED

Reply:
ah, I'm too late :(

Reply:
https://docs.docker.com/storage/volumes/

Reply:
Oops, will revert in another PR (and add a comment about it) and explain why in the issue. Good thing you caught it, thanks!